docker/tools.sh

```diff
@@ -14,7 +14,8 @@
 #
```

Comment:
Also merge this file in `scripts/ci/funcs.sh`
There is no need to have separated files for functions used by multiple builds.

Reply:
Again - didn't want to modify your scripts directly.
Ok.